test/data/Mock/RucioMockData.json

```diff
@@ -1404,6 +1404,7 @@
   "account": "ms-pileup"
  }],
  "evaluateRSEExpression": ["T2_XX_CERN"],
+ "evaluateRSEExpression:[('useCache', True)]": ["rse1", "rse2"],
```

Comment:
should we use useCache=False instead?

Reply:
We should use whatever signature of the call we have in place. Given that we are using the cache, it has to be True here as well.

Reply:
fine.